tools/protodoc/protodoc.py

```diff
@@ -132,7 +132,10 @@ def FormatCommentWithAnnotations(comment, type_name=''):
   if annotations.EXTENSION_ANNOTATION in comment.annotations:
     extension = comment.annotations[annotations.EXTENSION_ANNOTATION]
     formatted_extension = FormatExtension(extension)
-  return annotations.WithoutAnnotations(StripLeadingSpace(comment.raw) + '\n') + formatted_extension
+  formatted_extension_type = ''
+  if annotations.EXTENSION_TYPE_ANNOTATION in comment.annotations:
+    formatted_extension_type = "INFO ABOUT THE EXTENSION TYPE 8)"
```

Comment:
I assume this becomes more elaborated when you are done the draft work :)

Reply:
yep, hacking atm to see what is significant